src/test/groovy/ee/tuleva/onboarding/mandate/application/ApplicationServiceSpec.groovy

```diff
@@ -107,6 +110,24 @@ class ApplicationServiceSpec extends Specification {
       }
     }
     with(applications[3]) {
+      id == 123L
+      type == TRANSFER
+      status == PENDING
+      creationTime == ClockFixture.now
+      with(details) {
+        sourceFund.isin == "AE123232334"
+        exchanges.size() == 1
+        with(exchanges[0]) {
+          sourceFund.isin == "AE123232334"
+          targetFund == null
+          targetPik == "targetPik"
```

Comment:
I prefer to use "realistic" data in tests. It gives a better sense of what a PIK really should look like and makes the domain easier to understand for new engineers joining the team. Tests are effectively the _specification_ - and they should be real-life example inputs and real-life example outputs.

 For example, if it's an IBAN code then just generate a random IBAN: http://randomiban.com/?country=Estonia
e.g. `EE801281685311741971`